components/Paybutton/PaybuttonTrigger.tsx

```diff
@@ -133,23 +187,45 @@ export default ({ paybuttonId }: IProps): JSX.Element => {
                 {/* Only triggers if payment &gt; X */}
               </div>
               <div className={style.btn_row2}>
-                {(error === undefined || error === '') ? null : <div className={style.error_message}>{error}</div>}
-                {(successText === undefined || successText === '') ? null : <div className={style.success_message}>{successText}</div>}
+                {(posterError === undefined || posterError === '') ? null : <div className={style.error_message}>{posterError}</div>}
+                {(posterSuccessText === undefined || posterSuccessText === '') ? null : <div className={style.success_message}>{posterSuccessText}</div>}
                 <div>
-                  <button disabled={disableSubmit} type='submit' className='button_main'>{currentTriggerId === undefined ? 'Create' : 'Update'}</button>
+                  <button disabled={disablePosterSubmit} type='submit' className='button_main'>{currentPosterTriggerId === undefined ? 'Create' : 'Update'}</button>
                 </div>
                 <div>
                   {
-                    currentTriggerId !== undefined &&
-                      <button type='button' onClick={() => setClearModal(true)} className={style.delete_btn}>Clear</button>
+                    currentPosterTriggerId !== undefined &&
+                      <button type='button' onClick={() => setClearModal('poster')} className={style.delete_btn}>Clear</button>
+                  }
+                </div>
+              </div>
+            </div>
+            </form>
+          <form onSubmit={(e) => { void handleSubmitEmailTrigger(getSubmitTriggerHandler('email'))(e) }} method='post'>
+            <h5>Receive Email</h5>
+            <div>
+              <label htmlFor="emails">Receive email</label>(<span>{emailCredits} credits</span>)
```

Comment:
Maybe we can add another "what is this?" for the email credits?

![image](https://github.com/user-attachments/assets/225b0baa-191d-4610-8c1f-685eeab62e46)

Reply:
done, but with a mocked explanation

Reply:
I've had a change of heart on this.

Let's scrap the word "credits" in the UI and remove the "What is this?" helper and replace it with something that says:

> You may send up to 100 emails per day.

or

> You have sent X of a maximum of 100 emails per day.

We can eventually add a "need more?" button link thing instead of the "What is this?" one to allow them to upgrade their account to "Pro" or w/e we end up calling that.

Reply:
done, I changed "You have sent X of a maximum of 100 emails per day." to "You have sent X of a daily maximum of 100 emails.", let me know what you think.

Also created a dev3@paybutton.org user that has one button with a trigger and has not sent any email, to allow to test that too.